utils/build-script-impl

```diff
@@ -2028,6 +2051,12 @@ for host in "${ALL_HOSTS[@]}"; do
                     "${llvm_cmake_options[@]}"
                 )
 
+                if [[ "${USE_LLD_LINKER}" ]]; then
+                    cmake_options+=(
+                    -DLLVM_ENABLE_LLD:BOOL=YES
```

Comment:
Missing indentation?

Reply:
woops. yep.

Reply:
looks like that bad indentation was used in other places too. I'll fix it everywhere in an upcoming CL.